examples/cython_polar/setup.py

```diff
@@ -0,0 +1,18 @@
+from distutils.core import setup
+from distutils.extension import Extension
+from Cython.Build import cythonize
+
+codec = Extension(
+    name="codec",
+    sources=["codec.pyx"],
+    libraries=["aff3ct-2.3.5-384-gec40f26"],
+    library_dirs=["../../lib/aff3ct/build/lib"],
+    include_dirs=["../../lib/aff3ct/include", "../../lib/aff3ct/lib/cli/src", "../../lib/aff3ct/lib/MIPP/src", "../../lib/aff3ct/lib/MIPP/src", "../../lib/aff3ct/lib/rang/include"],
+    language="c++",
+    extra_compile_args=["-std=c++14"],
+    extra_link_args=["-std=c++14"]
+)
+setup(
+    name="codec",
+    ext_modules=cythonize([codec])
+)
```

Comment:
I think that this file should be generated from the `CMakeLists.txt` file.

Reply:
Nit.

Reply:
Done.

Reply:
This is better than the previous version but it will not work if AFF3CT is installed elsewhere (on the system for instance).
I really think that the values in these variables should be set from the `CMakeLists.txt` file. I will check tomorrow to give you an example of what I mean :-).

Reply:
Thank you. TBH I do not know how to generate this `setup.py` in the `CMakeLists.txt` file. Yes, please help me with an example.